sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/NetworkStateHelper.java

```diff
@@ -149,81 +150,73 @@ private IntentFilter getOldIntentFilter() {
      * @return true for connected, false for disconnected.
      */
     public synchronized boolean isNetworkConnected() {
-        return mNetworkType != null || !mAvailableNetworks.isEmpty();
+        return mConnected || isAnyNetworkConnected();
     }
 
     /**
-     * Handle network available update on API level >= 21.
+     * Check if any network is connected.
```

Comment:
Maybe comment it's used only as a fallback.

Reply:
https://github.com/Microsoft/AppCenter-SDK-Android/pull/943/files#diff-f8b58324d3d38f4997ba0df305858c37R216